R/zzz.R

```diff
@@ -1,4 +1,4 @@
 .onLoad <- function(...) {
-  options(warnPartialMatchDollar = TRUE)
+  options(warnPartialMatchDollar = TRUE, warn = -99)
```

Comment:
What does that mean?

Reply:
This is outdated. That's what I meant when I said suppressing warnings globally.

We don't use that trick anymore.